jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
The formatting here is a bit odd. 

Do you mind copying the jest config from here: https://github.com/open-rpc/generator-client/blob/master/jest.config.js

We can remove the coverage, though. =P